### PR TITLE
336362 new journeys part v

### PR DESCRIPTION
**Pull request by feedmypixel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,6 +51,26 @@ To run the application in `development` mode run:
...
+### Aws localstack
+
+#### Install and run LocalStack AWS
+
+- Install [LocalStack AWS CLI](https://github.com/localstack/awscli-local)
+- Run AWS LocalStack Docker container:
+
+```bash
+docker run --pull=always -d -p 4566:4566 -p 4510-4559:4510-4559 localstack/localstack:latest
+```
```
Comment (flurdy) [inline README.md:63]: I have run both, but just assumed people will run the one in cdp-local-environment. But I guess for generic tenant development using the raw localstack is best to show as an example.

File: README.md [hunk 2]
```diff
@@ -48,6 +51,26 @@ To run the application in `development` mode run:
...
+
+#### Setup local buckets
+
+You need local buckets setup in localstack
+
+```bash
+awslocal s3 mb s3://cdp-uploader-quarantine --endpoint-url http://localhost:4566
+awslocal s3 mb s3://my-bucket --endpoint-url http://localhost:4566
```
Comment (flurdy) [inline README.md:71]: Are you not missing a `=`? after the alias. Also if using localstack/localstack the default endpoint is https, whilst with cdp-local-env it is http

Comment (flurdy) [reply README.md:71]: (ignore the `=` comment. Removed it)

Comment (flurdy) [reply README.md:71]: Though if using awslocal the endpoint-url is superfluous. As all it does is append it to the raw `aws` command

Comment (feedmypixel) [reply README.md:71]: nope - all ok - https://docs.localstack.cloud/user-guide/aws/mwaa/#create-a-s3-bucket

File: README.md [hunk 3]
```diff
@@ -48,6 +51,26 @@ To run the application in `development` mode run:
 npm run dev
 ```
 
+### Aws localstack
+
+#### Install and run LocalStack AWS
+
+- Install [LocalStack AWS CLI](https://github.com/localstack/awscli-local)
```
Comment (flurdy) [inline README.md:58]: Could add install localstack CLI https://docs.localstack.cloud/getting-started/installation/#localstack-cli

Comment (flurdy) [reply README.md:58]: Also I would not recommend the awscli-local. As it needs AWS CLI v1 which is old.

Comment (flurdy) [reply README.md]: I just add an alias, in fish not bash
```
function awslocal
  aws --profile localstack $argv --endpoint-url http://localhost:4566
end
```
I can see if change that separately in the readme :)

**Pull request merged**